code/__DEFINES/misc.dm

```diff
@@ -178,6 +178,7 @@
 #define ZLEVEL_SPACEMAX 7
 #define ZLEVEL_MINING 5
 #define ZLEVEL_SPACEMIN 3
+#define ZLEVEL_DEEPSPACE 6
```

Comment:
I'll just remove this line anyway since I did this in the weather PR and dont want it conflict